Software/Serial_read/py_arduino_serial_GUI.py

```diff
@@ -182,6 +182,7 @@ def __init__(self):
         trialSettingsLayout = QFormLayout()
 
         self.num_trials = SpinBox(trialSettingsLayout, "Number of trials", 6, step=1, min=1, max=6)
+        self.is_training = CheckBox(trialSettingsLayout, "Is training session?")
```

Comment:
I see it gets checked by default when running the GUI. I feel it makes sense to have it unchecked by default? the checkbox class could have a `checked` parameter. I can add this logic in my new PR